### PR TITLE
add meet-up guidelines to /community

### LINKED ISSUE #3352: Add meet-up guidelines to /community
We want to bring more visibility to the meet-up guidelines [[1](https://www.notion.so/gitpod/Hosting-a-Gitpod-Community-meet-up-77bdde81c5b04d80b93e425ebdd568ef)] on `/community` page [[2](https://www.notion.so/gitpod/Gitpod-Paris-Retro-6f50d5b84f584b128c3c34d8acc9ac39#69b9001bef054cd3a4d763894714463b)]. 

My proposal is we create another card under this heading with the following copy:

2023-02-01 at 16 09 07 - Google Chrome@2x

## Host a Gitpod Community meet-up

Gitpod Community meet-ups are organized by passionate community members in their local area. They are an opportunity to connect with others in the community and learn and share the best Gitpod practices and knowledge. 

**Button 1:** Hosting guidelines (https://www.notion.so/gitpod/Hosting-a-Gitpod-Community-meet-up-77bdde81c5b04d80b93e425ebdd568ef)

**Button 2:** View playlist (https://www.youtube.com/playlist?list=PL3TSF5whlprWHSTPbBrpGyxetRMOgUTd5)

From a design POV, are you happy with this @chrifro?

### PR DESCRIPTION
**Pull request by ShashwatNautiyal**
## Description
Add meet-up guidelines to the community page.

### Screenshot
image

## Related Issue(s)

Fixes #3352 

## How to test
Go to  [cummunity](https://deploy-preview-3385--gitpod-kumquat.netlify.app/community) page

## Release Notes

```NONE```

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3385"></a>

### CODE REVIEW

File: src/lib/components/community/callout-boxes.svelte [hunk 1]
```diff
@@ -54,6 +54,26 @@
...
+          href: "https://www.notion.so/gitpod/Hosting-a-Gitpod-Community-meet-up-77bdde81c5b04d80b93e425ebdd568ef",
+        },
+        ctaButton: {
+          text: "View playlist",
+          href: "https://www.youtube.com/playlist?list=PL3TSF5whlprU4vRoYBK6-QhFAbwAsoKmu",
+        },
+        image: {
+          src: "/images/community/meet-up.png",
+          alt: "Gitpod community meet-up",
+          classNames: "rounded-2xl",
```
Comment (Siddhant-K-code) [inline src/lib/components/community/callout-boxes.svelte:73]: Thanks for the PR 🙏🏽 

A small nit, card image is looking bigger than what was in the [expected image](https://github.com/gitpod-io/website/issues/3352#issuecomment-1418626375):

|Expected|Actual|
|:---:|:---:|
|image|image|

File: src/lib/components/community/callout-boxes.svelte [hunk 2]
```diff
@@ -54,6 +54,26 @@
...
+        moreButton: {
+          text: "Hosting guidelines",
+          href: "https://www.notion.so/gitpod/Hosting-a-Gitpod-Community-meet-up-77bdde81c5b04d80b93e425ebdd568ef",
+        },
+        ctaButton: {
+          text: "View playlist",
+          href: "https://www.youtube.com/playlist?list=PL3TSF5whlprU4vRoYBK6-QhFAbwAsoKmu",
+        },
+        image: {
+          src: "/images/community/meet-up.png",
```
Comment (Siddhant-K-code) [inline src/lib/components/community/callout-boxes.svelte:71]: **Suggestion**: Please consider using `.webp` image. You can easily convert it by following [our guide](https://github.com/gitpod-io/website/blob/main/docs/adr/0003-optimal-assets-format.md#convert-image-asset-to-webp)

**Review by Siddhant-K-code — APPROVED** [review]
Code LGTM. 

/hold for review from @chrifro

**Review by chrifro — COMMENTED** [review]
The image should have the same width as the community heroes image on the first card. 

Bildschirm­foto 2023-02-07 um 09 02 08

Feel free to merge it once that's fixed 🙌

File: src/lib/components/community/callout-boxes.svelte (hunk 1 above)
Comment (ShashwatNautiyal) [reply src/lib/components/community/callout-boxes.svelte]: > Thanks for the PR 🙏🏽
> 
> A small nit, card image is looking bigger than what was in the [expected image](https://github.com/gitpod-io/website/issues/3352#issuecomment-1418626375):
> 
> Expected	Actual
> image	image

@chrifro Can you take a look at the expected image? Referring to #3352, The image is much smaller than the community heroes' image on the first card.

I will revert my changes if the image doesn't need to be the exact dimensions according to the design in #3352 

Cc: @Siddhant-K-code

**Comment by ShashwatNautiyal** [comment]
> The image should have the same width as the community heroes image on the first card.
> 
> Bildschirm­foto 2023-02-07 um 09 02 08
> 
> Feel free to merge it once that's fixed 🙌

Please refer to this [discussion](https://github.com/gitpod-io/website/pull/3385#discussion_r1098447090)

**Comment by ShashwatNautiyal** [comment]
Updated 👇🏻

### Screenshot
image

**Pull request merged**